modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -27,6 +27,18 @@ This illustration summarizes how each asset type (in green) appears in Exchange:
 
 image::ex2-exchange-assets.png[Illustration - How Assets appear in Exchange]
 
+== Asset Limits
+
+Exchange uses an asset limit to prevent accidental or intentional denial of service by uploading too many assets.
+
+The limit is 500 assets for organizations with trial accounts, and 100,000 assets for other organizations.
```

Comment:
I would change `100,000` to `100.000`, what do you think?

Reply:
I'm glad you brought this up. The US uses "100.000" to mean one hundred but many other counties use it to mean one hundred thousand. I'll follow the advice from Wikipedia and change this to "100 000 (one hundred thousand)".

https://en.wikipedia.org/wiki/Decimal_separator